mmx_sale_triple_price/README.rst

```diff
@@ -0,0 +1,98 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
check https://github.com/Elico-Corp/openerp-7.0/pull/92
Most of my remarks should apply. Then ping me